src/main/java/org/graylog2/syslog4j/impl/unix/socket/UnixSocketSyslog.java

```diff
@@ -39,6 +40,12 @@ public void setSunPath(String sunPath) {
             System.arraycopy(sunPath.getBytes(), 0, this.sun_path, 0, sunPath.length());
             System.arraycopy(ZERO_BYTE, 0, this.sun_path, sunPath.length(), 1);
         }
+
+		@Override
+		protected List getFieldOrder() {

```

Comment:
What do you mean default? there is no default implementation (it is an
abstract method) .. protected abstract List getFieldOrder();

On Wed, Feb 17, 2016 at 4:48 PM, Jochen Schalanda <notifications@github.com>
wrote:

> In
> src/main/java/org/graylog2/syslog4j/impl/unix/socket/UnixSocketSyslog.java
> <https://github.com/Graylog2/syslog4j-graylog2/pull/6#discussion_r53183092>
> :
>
> > @@ -39,6 +40,12 @@ public void setSunPath(String sunPath) {
> >              System.arraycopy(sunPath.getBytes(), 0, this.sun_path, 0, sunPath.length());
> >              System.arraycopy(ZERO_BYTE, 0, this.sun_path, sunPath.length(), 1);
> >          }
> > +
> > +		@Override
> > +		protected List getFieldOrder() {
>
> Is there a problem with the default implementation of this method?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Graylog2/syslog4j-graylog2/pull/6/files#r53183092>.
>